sections/04_study.md

```diff
@@ -82,7 +82,97 @@ use interesting network architectures to approach single-cell data.
+binning and classification of species, there is functional identification and
+annotation of sequence reads [Yok,Soueidan]. However, the focus on
+taxonomic/functional annotation is just the first step.  Once organisms are
+identified, there is the interest in understanding the interrelationship
+between these organisms and host/environment phenotypes [Guetterman].  One of
+the first attempts was a survey of supervised classification methods for
+microbes->phenotype classification [Knights], followed by similar studies
+that are more massive in scale [Stratnikov, Segata].  There have been
+techniques that bypass the taxonomic classification step altogether [Ding]
+(sequence composition->phenotype classification).  Also, researchers have
+looked into how feature selection can improve classification [Liu, Segata],
+and techniques have been proposed that are classifier-independent
+[Ditzler,Ditzler].
+
+So, how have neural networks been of use?  One of the first techniques of
+genome binning used neural networks [Abe]. Neural networks can be slow, and
+so far only TAC-ELM [tac-elm] has utilized it to taxonomically classify
+massive amounts of metagenomic data.  Also, neural networks can fail to
+perform if there are not enough training examples, which is the case with
+taxonomic classification (since only ~10% of estimated species have been
+sequenced).  For “de novo” genome binning, Essinger et al. use ART, a neural
+network algorithm called Adaptive Resonance Theory, to cluster similar
+genomic fragments and showed that it has better performance than K-means; 
+however, other methods based on interpolated Markov models [Salzberg] have
+performed better.  Due to the complexity of the problem, neural networks have
+been applied more to gene annotation (e.g. Orphelia [Hoff]), which usually
+have plenty of training examples.  Representations (similar to Word2Vec [ref]
+in natural language processing) for protein family classification has been
+introduced and classified with a skip-gram neural network [Asgari]. 
+Recurrent neural networks show good performance for homology and protein
+family identification [Hochreiter, Sonderby].  Interestingly, Hochreiter, who
+invented Long Short Term Memory, delved into homology/protein family
+classification in 2007, and therefore, deep learning is deeply rooted in
+functional classification methods.
+
+Most neural networks are being used for short sequence->taxa/function
+classification, where there is a lot of data for training (and thus suitable
+for NNs).  And, as a short side note, recurrent neural networks are showing
+success for base-calling (and thus removing the large error in the
+measurement of a pore's current signal) for the relatively new Oxford
+Nanopore sequencer [Boza].  However, due to small nubmers of metagenomic
+samples in studies, neural network uses for classifying phenotype from
+microbial composition are just beginning.   A standard MLP was able to
```

Comment:
Not sure whether we'll have defined MLP previously.  @cgreene will that be in the intro?